### PR TITLE
add new game feature prototype

### PR DESCRIPTION
**Pull request by exosty**
At least something works, but still a lot to do

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -1,4 +1,23 @@
 class GamesController < ApplicationController
   def index
+    @games = Game.all
```
Comment (vladimirbazhanov) [inline app/controllers/games_controller.rb]: please add before_filter with authorize_user! devise helper and user current_user.games.all here?

File: app/controllers/games_controller.rb [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+  end
+
+  def new
+    @game = Game.new
```
Comment (vladimirbazhanov) [inline app/controllers/games_controller.rb]: current_user.games.new may be better

File: app/controllers/games_controller.rb [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+  end
+
+  def create
+    @game = Game.new(params[:game])
```
Comment (vladimirbazhanov) [inline app/controllers/games_controller.rb]: current_user.... you know)

File: app/controllers/games_controller.rb [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+    @game.player_1 = current_user.id
```
Comment (vladimirbazhanov) [inline app/controllers/games_controller.rb]: you can make something like 

class Game
  belongs_to player_1, class_name: User, foreign_key: 'player_1'

File: app/models/round_qestion.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class RoundQestion < ActiveRecord::Base
```
Comment (vladimirbazhanov) [inline app/models/round_qestion.rb]: RoundQuestion, pls correct typo

File: spec/models/round_question_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe RoundQuestion, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (vladimirbazhanov) [inline spec/models/round_question_spec.rb]: or add real tests or delete this file pls

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (exosty) [inline app/controllers/games_controller.rb]: u mean authenticate_user! helper?

**Comment by vladimirbazhanov** [comment]
@exosty я смержу ваш ПР чтобы немного поправить и доделать, результат выложу скоро

**Pull request merged**